src/backend/distributed/planner/query_pushdown_planning.c

```diff
@@ -1390,13 +1331,13 @@ RelationInfoContainsRecurringTuples(PlannerInfo *plannerInfo, RelOptInfo *relati
 		RangeTblEntry *rangeTableEntry = plannerInfo->simple_rte_array[relationId];
```

Comment:
I'm kind of wondering why we're not using bms_next_member without the additional bms_copy step.

Reply:
I don't think there is a specific reason. It is likely to be a copy & paste from pg code, where bms_copy is required. Let me remove them in a separate commit as well

Reply:
Wait, quite a lot of tests fails if I remove those :) I'll check what's the reason

Reply:
It turns out that the API we use modifies the bitmap: https://github.com/postgres/postgres/blob/05d8449e73694585b59f8b03aaa087f04cc4679a/src/backend/nodes/bitmapset.c#L981-L994

Reply:
https://github.com/citusdata/citus/pull/4230